assets/css/editor-blocks.css

```diff
@@ -1,4 +1,4 @@
y:0}to{opacity:1}}.travel-object-cover,.travel-object-cover>img{-o-object-fit:cover;object-fit:cover}amp-img.rounded img{border-radius:4px}amp-img.circle img{border-radius:50%}.travel-no-focus:focus{outline:none}.travel-type-active,.travel-type-active:visited{color:#fd68ae}.travel-type-artistic,.travel-type-artistic:visited{color:#32dada}.travel-type-drinks,.travel-type-drinks:visited{color:#4e93f8}.travel-type-fashion,.travel-type-fashion:visited{color:#f1c363}.travel-type-food,.travel-type-food:visited{color:#aa5efd}.travel-type-music,.travel-type-music:visited{color:#ec8936}.travel-type-nature,.travel-type-nature:visited{color:#14b52a}.travel-type-nightlife,.travel-type-nightlife:visited{color:#e83a3a}.travel-type-tours,.travel-type-tours:visited{color:#0ece9e}.travel-type-water,.travel-type-water:visited{color:#3abee8}.travel-overflow-container{display:inline-block;padding-left:calc((100vw - 72rem) / 2);padding-right:calc((100vw - 72rem) / 2)}.travel-link,.travel-link:visited{color:#8b58e3;text-decoration:none;font-family:inherit;-webkit-transition:color .15s ease-in;transition:color .15s ease-in}.travel-link:hover{color:#b99aef}.travel-link:focus{outline:none;border-bottom:2px solid rgba(50,58,67,.2)}.travel-link.gray,.travel-link.gray:visited{color:#8f98a3}.travel-link.gray:hover{color:#b4b4b4}.travel-pill{border-radius:1em;padding:2px 8px;color:#fff;background-color:#fb52a2}.travel-shadow,.travel-shadow-hover{box-shadow:0 .4rem 1.1rem rgba(50,58,67,.12)}.travel-shadow-hover{-webkit-transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,transform .25s;transition:box-shadow .25s,transform .25s,-webkit-transform .25s}.travel-shadow-hover:not([disabled]):focus,.travel-shadow-hover:not([disabled]):hover{-webkit-transform:scale(1.02) translateY(-2%);transform:scale(1.02) translateY(-2%);box-shadow:0 .4rem 2rem rgba(50,58,67,.4)}.h1.line-height-2,.h2.line-height-2,.h3.line-height-2,.h4.line-height-2,.h5.line-height-2,.h6.line-height-2,h1.line-height-2,h2.line-height-2,h3.line-height-2,h4.line-height-2,h5.line-height-2,h6.line-height-2{line-height:1.4}.ampstart-btn{-webkit-transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,-webkit-transform .25s;transition:box-shadow .25s,transform .25s;transition:box-shadow .25s,transform .25s,-webkit-transform .25s;line-height:1.125rem;padding:.5rem 1rem;border-color:transparent}.ampstart-btn[disabled]{opacity:.5;cursor:default}.ampstart-btn:not([disabled]):hover{-webkit-transform:scale(1.02) translateY(-2%);transform:scale(1.02) translateY(-2%);box-shadow:0 .4rem 2rem rgba(50,58,67,.4)}.ampstart-btn:not([disabled]):focus{outline:none;box-shadow:none}.ampstart-btn.bold{font-weight:600}.travel-date-input{line-height:1.125rem;padding:.5rem 1rem;padding-left:calc(.5rem * 2.5 + 1.2em)}.travel-date-input .travel-icon{position:absolute;left:0;top:0;box-sizing:content-box;padding:.5rem 1rem}.travel-date-input.input-dark .t
```

Comment:
The diff here is much simpler than it looks. I simply removed the style rule `min-width:100vw` from `.travel-overflow-container`.